src/server/response.cpp

```diff
@@ -146,17 +146,17 @@ std::unique_ptr<ContentResponse> ContentResponseBlueprint::generateResponseObjec
   return r;
 }
 
-HTTPErrorHtmlResponse::HTTPErrorHtmlResponse(const InternalServer& server,
-                                             const RequestContext& request,
-                                             int httpStatusCode,
-                                             const std::string& pageTitleMsgId,
-                                             const std::string& headingMsgId,
-                                             const std::string& cssUrl)
+HTTPErrorResponse::HTTPErrorResponse(const InternalServer& server,
```

Comment:
We must make sure that the XML response is handled like a raw response (i.e. not postprocessed in any way).